src/components/form/SelectCondition/index.tsx

```diff
@@ -53,35 +43,29 @@ export const SelectCondition = () => {
 
   return (
     <>
-      <Row cols={'1fr'} marginBottomXL>
-        <TitleValue
-          title="Condition Id"
-          titleControl={<TitleControl onClick={openModal}>Select Condition</TitleControl>}
-          value={
-            <>
-              <Textfield
-                error={!!errors.length}
-                onChange={imputHandler}
-                placeholder="Please select a condition..."
-                type="text"
-                value={conditionId}
-              />
-              {loading && (
-                <ErrorContainer>
-                  <InfoMessage>Loading...</InfoMessage>
-                </ErrorContainer>
-              )}
-              {!!errors.length && (
-                <ErrorContainer>
-                  {errors.map((error, i) => (
-                    <Error key={`error${i}`}>{error}</Error>
-                  ))}
-                </ErrorContainer>
-              )}
-            </>
-          }
-        />
-      </Row>
+      <TitleValue
+        title={title}
+        titleControl={<TitleControl onClick={openModal}>Select Condition</TitleControl>}
+        value={
+          <>
+            <Textfield
+              error={!!errors.length}
+              onChange={imputHandler}
+              placeholder={loading ? 'Loading...' : 'Please select a condition...'}
```

Comment:
I'd thought about this but the problem I found is when the user manually enters a condition id. 
I think the component should show some hint to the user to let it know data is being fetched/validated so a placeholder wont work in this case. Right?

Reply:
Good point. I think we should improve this through some kind of new design.

I'll create an issue for that, and then implement it.

Reply:
Here's the issue https://github.com/gnosis/conditional-tokens-explorer/issues/139